helpers/remote_rooms/remote_rooms_location.py

```diff
@@ -0,0 +1,11 @@
+
+def map_remote_room_location_to_filter():
+    '''
+        map filters for room location
+    '''
+    return {
+            'Nairobi': lambda remote_room: remote_room.startswith('Nairobi - '),
+            'Lagos': lambda remote_room: remote_room.find('-ET-') != -1,
+            'Kampala': lambda remote_room: remote_room.find('Kampala') != -1
+            and not remote_room.startswith('Nairobi'),
+        }
```

Comment:
@verenceLola, how about New York rooms? Also, what happens when the person who creates the rooms changes the string format? We are dealing with unstructured data, how will you ensure this filter does not break?

Reply:
I had a sync with @Bonifase and we came up with the above structure for the filters since there was no other means to determine the location of a given room. The data returned from the calendar API doesn't provide any other means of determining a room's location.

Reply:
@verenceLola we've got some confirmation from the product team that they will be consistent in naming these rooms. We can't trust humans however, so I suggest you leave filter functionality as default but add a flag which when passed, returns all rooms.

Reply:
> I've tested it and it works well @verenceLola, the only concern is that there's no standard way the rooms are created such that it's easier to identify the location, hence some rooms might miss from a list of rooms in specific location despite existing in that location.

After consultation with the product team, a standardised way of naming the rooms has been adopted. Our filter system should work okay. refer to [this](https://github.com/andela/mrm_api/pull/268#discussion_r261851161)